1/step2_2_sort.py

```diff
@@ -0,0 +1,17 @@
+from typing import List
+
+
+class Solution:
+    def twoSum(self, nums: List[int], target: int) -> List[int]:
+        # (値, 元インデックス) のタプルリストを作成
+        nums_with_index = [(num, i) for i, num in enumerate(nums)]
+        nums_with_index.sort(key=lambda x: x[0])
```

Comment:
これは好みの範囲ですが、itemgetterを使っても書けます。

```suggestion
        from operator import itemgetter
        nums_with_index.sort(key=itemgetter(0))
```

https://docs.python.org/ja/3.13/library/operator.html#operator.itemgetter

Reply:
tupleの比較はデフォルトで前から順に行われるのでkeyは何も指定しない、というのもありかなと思いました

Reply:
ご指摘ありがとうございます、key指定なしも許容されるのですね。tuple比較の挙動を理解していなかったので、勉強になりました。